src/applications/simple-forms/mock-simple-forms-patterns/config/form.js

```diff
@@ -242,6 +243,13 @@ const formConfig = {
           schema: dynamicFields.schema,
           depends: includeChapter('miscellaneous'),
         },
+        addressArray: {
```

Comment:
if we include this, can we use the same naming convention as the other array pages, and move it to the similar arraySinglePage page array part? (There are 3 other area areas on this form).

Also thanks for testing this, it looks like using address in a single page array like this is actually full of bugs (required fields not showing up, labels not appearing, basically the updateSchema code is not running, which sets all those fields right), which seems to be preexisting bugs unrelated to your code. Actually because of that, I'm hesitant about including it in this form, since this form is supposed to be "good examples to look at". 

How about we don't include these pages for now, and we make a ticket to look into these issues? (I can do that), and this will have served its purpose as testing and bug discovery.

Reply:
I moved this to singlePageArray and commented it out.